posthog/temporal/tests/test_squash_person_overrides_workflow.py

```diff
@@ -795,7 +790,13 @@ def organization_uuid(query_inputs, django_db_setup_fixture):
     """
     organization_uuid = uuid4()
 
-    with psycopg2.connect(query_inputs.postgres.database_url) as connection:
+    with psycopg2.connect(
+        dbname=settings.DATABASES["default"]["NAME"],
+        user=settings.DATABASES["default"]["USER"],
+        password=settings.DATABASES["default"]["PASSWORD"],
+        host=settings.DATABASES["default"]["HOST"],
+        port=settings.DATABASES["default"]["PORT"],
```

Comment:
can we put this into a function, so there's less copy-pasta and it's easier to change in the future?